frontend/src/components/Options.vue

```diff
@@ -56,8 +125,25 @@ export default {
 
       Events.$emit('setting:hideAdvanced', { value: this.hideAdvanced });
     },
-  },
-};
+    async onClaimTokens() {
+      if(this.canClaimTokens) {
+        await this.claimTokenRewards();
+      }
+    },
+    async claimSkill(stage: number) {
+      if(stage === 0) {
```

Comment:
can this be an enum instead?

Reply:
Done! It's working as is, but should I have declared the enum somewhere else or anything?

Reply:
since the enum is just for this page, having it only on this page is fine